packages/components/pie-cookie-banner/test/component/pie-cookie-banner.spec.ts

```diff
@@ -79,14 +79,14 @@ test.describe('PieCookieBanner - Component tests', () => {
         const cookieBanner = await page.locator(componentSelector);
```

Comment:
Might have missed it as i commented late on your pr (sorry). These awaits can go i think.